handlers/_swagger.js

```diff
@@ -20,7 +20,7 @@ var polymorphicValidation;
 exports.init = function (app, auth, config, logger, serviceLoader, swagger, callback) {
     var cfg = config.get('swagger');
     responseModelValidationLevel = /error|warn/.test(cfg.validateResponseModels) ? cfg.validateResponseModels : 0;
-    polymorphicValidation = cfg.polymorphicValidation;
+    polymorphicValidation = _.get(cfg, 'polymorphicValidation', true);
```

Comment:
ha! just thought of (what I think is) a better way to do this (because it means that no matter what the user puts in the config we'll get a boolean value), so one more commit please (unless I'm wrong, then let me know):
```javascript
polymorphicValidation = (cfg.polymorphicValidation !== false);
```